pkg/config/config.go

```diff
@@ -60,8 +60,11 @@ func LoadConfig() *Config {
 
 	f, err := os.Open(fullConfigFile)
 	if err != nil {
-		createDefaultConfig(fullConfigFile)
-		return nil
+		f, err = createDefaultConfig(fullConfigFile)
```

Comment:
the handle returned by createDefaultConfig is open for writing, but here we'll try to read it, how does this work?

Reply:
It's opened with read/write permissions. We could close that handle and open it read only before returning.

Reply:
I didn't know os.Create used O_RDWR